git_tools/git_utils.py

```diff
@@ -175,3 +215,26 @@ def get_current_branch(repo_path):
 def apply_patch(repo_path, patch_path):
     '''Apply a patch to the given repo.'''
     _execute_git_command(repo_path, "cat \"{}\" | git am -3 -k".format(patch_path))
+
+def fill_in_params(filename, params=None, **kwargs):
```

Comment:
This probably could have been done with something like jinja instead of writing this your self. However, since it is for a specific use case maybe that would have been overkill.

Reply:
Yeah, it may get weird putting jinja inside of patches too. If we ever need to have this do more stuff or be more robust, we could switch it over.